mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -3015,16 +3014,16 @@ def _compute_evaluation_dataset_digest(self, name: str, last_update_time: int) -
     def create_evaluation_dataset(
         self,
         name: str,
-        tags: Optional[dict[str, str]] = None,
-        experiment_ids: Optional[list[str]] = None,
+        tags: dict[str, str] | None = None,
+        experiment_id: list[str] | None = None,
```

Comment:
Shouldn't we keep this argument as `experiment_ids` as this is a list of experiment id?